src/app/page.tsx

```diff
@@ -1,20 +1,46 @@
-import { getAuthors } from '@/app/_api/(authors)/getAuthors'
-import { getPosts } from '@/app/_api/getPosts'
+'use client'
+
 import BlogList from '@/app/_components/BlogList'
-import { Post, User } from '@/types'
-import { Spacer, Text } from '@kuma-ui/core'
+import SearchBar, { filterSearched } from '@/app/_components/SearchBar'
+import { useAppStore } from '@/helpers/store'
+import { Heading } from '@chakra-ui/react'
+import { useState } from 'react'
+import useAsyncEffect from 'use-async-effect'
+
+const handleError = (e: any) => {
+  console.error("couldn't fetch posts or authors :", e)
+  throw e
+}
```

Comment:
Deserve it's own file, and a toast to show the message. Do not rethrow

Reply:
toast created